src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -196,20 +197,20 @@ private void ProcessFileContent(string path)
                         if (count > Count)
                         {
                             bytesRead -= (int)(count - Count);
-                            WriteHexidecimal(buffer.Slice(0, bytesRead), path, offset);
+                            WriteHexadecimal(buffer.Slice(0, bytesRead), path, offset);
                             break;
                         }
 
-                        WriteHexidecimal(buffer.Slice(0, bytesRead), path, offset);
+                        WriteHexadecimal(buffer.Slice(0, bytesRead), path, offset);
 
                         offset += bytesRead;
                     }
                 }
             }
-            catch (IOException ioException)
+            catch (IOException fileException)
```

Comment:
It is not always true :-) Please revert.

Reply:
CodeFactor complained that the `io` prefix looks too much like a Hungarian prefix, from memory. Is this simply not a concern?

Reply:
We frequently use `catch (IOException e)`.
(I guess we will disable the check.)